src/cli/abrtcli/cli.py

```diff
@@ -173,10 +173,17 @@ def list_problems(args):
     else:
         print(_('No problems'))
 
+    settings = problem.load_conf_file('abrt.conf')
+    if settings.get('AutoreportingEnabled', 'no') in ['disabled', 'no', '0', 'off']:
```

Comment:
One more thing: 
https://github.com/abrt/abrt/blob/d10caaf40e613b566430c2b015d604b4da9e65d1/abrt.spec.in#L522

https://github.com/abrt/abrt/blob/d10caaf40e613b566430c2b015d604b4da9e65d1/configure.ac#L279

Reply:
Removed them. The code can just be patched out on RHEL.